doc/articles/getting-started-tutorial-1.md

```diff
@@ -41,6 +41,8 @@ The tutorial walks you through creating a cross platform application with Uno, w
     - Click back on the Browse tab and install the following NuGet Packages to each of the projects in your solution:
         - `Refractored.MvvmHelpers`
 
+1. Finally, you'll need to close any opened file in the editor, then close the solution or Visual Studio, then re-open it. This is a workaround for a Visual studio issue regarding the XAML editor.
```

Comment:
Maybe it will be helpful also to precise that without that it's impossible to select UWP in the dropdown (left top corner) to be able to have the full XAML intellisense experience.

Reply:
That's probably too much detail for a tutorial... Let's see if people ask questions about it.

Reply:
Ok ;)